src/common/snippets/include/snippets/runtime_configurator.hpp

```diff
@@ -110,7 +122,8 @@ class RuntimeConfigurator {
     /**
      * @brief Calculate data offsets of LinearIR and update these values in RuntimeConfig
      */
-    void update_data_offsets() const;
+    void update_data_offsets(const std::vector<ov::snippets::VectorDims>& shapes = {},
+                             const std::vector<std::vector<size_t>>& layouts = {}) const;
```

Comment:
Initially, there was an idea to make `update_data_offsets` static function. However, it will require 6 parameters, so the signature will be cumbersome